src/material-experimental/mdc-slider/slider.ts

```diff
@@ -236,6 +238,17 @@ export class MatSliderVisualThumb implements AfterViewInit, OnDestroy {
   }
 }
 
+/**
+ * Provider Expression that allows slider thumb inputs to register as a ControlValueAccessor.
+ * This allows it to support [(ngModel)] and [formControl].
+ * @docs-private
+ */
+export const MAT_SLIDER_THUMB_VALUE_ACCESSOR: any = {
```

Comment:
You don't need to export this constant, you should be able to put it inline in the `providers` expression.